lib/fizzy/execute.cpp

```diff
@@ -753,4 +753,13 @@ execution_result execute(const Module& module, FuncIdx function, std::vector<uin
     auto instance = instantiate(module);
     return execute(instance, function, args);
 }
+
+FuncIdx find_exported_function(const Module& module, std::string_view name)
+{
+    for (auto const& export_ : module.exportsec)
+        if (export_.type == ExportType::Function && name == export_.name)
+            return export_.index;
+
+    throw std::runtime_error("Exported function not found.");
```

Comment:
I think this function should return `optional` given it is an external API used prior to execution.

Reply:
changed